### ISSUE TITLE
[Frage] Bitte um Prüfung: Haben gleichnamige Seitenbereiche denselben Zweck?

### ISSUE THREAD

**Issue by AndiWebred** [RRZE-Webteam/FAU-Elemental#693]
### Bereich

Frontend

### URL zur betroffenen Seite (optional)

https://www.fau.de

### Ihre Frage

In Siteimprove haben wir einen potentiellen Fehler, bei dem ich mir nicht sicher bin, ob das einer ist: "Haben gleichnamige Seitenbereiche denselben Zweck?"
Das betrifft in einem Fall alle Seiten, nämlich die vier Navigationsmenüs: Servicemenü, Hauptmenü, Zielgruppenmenü, Footermenü.
Hier die Erläuterung von Siteimprove dazu: https://help.siteimprove.com/support/solutions/articles/80001058803-accessibility-rule-page-sections-with-the-same-name-do-not-serve-the-same-purpose-explained

Und hier der Link zur Siteimprove-Detail-Seite für die Startseite: https://my2.siteimprove.com/Inspector/705695/A11Y/Page?pageId=74034471241&impmd=CDF7C5F66B15D04FC961306C8B003AD3&expand=potential&issue=cantTell+failed&wcag=twopointtwo#/sia-r55/cantTell/M3dXeUUvcGI=;MjI2ZTJmYTA= 

Würdet ihr das mal prüfen, ob das aus eurer Sicht unproblematisch oder problematisch ist?

### Screenshot(s) (optional)

```image

```
